src/healthkit.cs

```diff
@@ -2043,17 +2045,20 @@ interface HKWorkoutEvent : NSSecureCoding, NSCopying {
 		[Wrap ("WeakMetadata")]
 		HKMetadata Metadata { get; }
 
-		[Availability (Deprecated = Platform.iOS_11_0 | Platform.Watch_4_0, Message = "Use 'Create (HKWorkoutEventType, NSDateInterval, HKMetadata)' instead.")]
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'Create (HKWorkoutEventType, NSDateInterval, HKMetadata)' instead.")]
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use 'Create (HKWorkoutEventType, NSDateInterval, HKMetadata)' instead.")]
 		[Static, Export ("workoutEventWithType:date:")]
 		HKWorkoutEvent Create (HKWorkoutEventType type, NSDate date);
 
-		[Availability (Introduced = Platform.iOS_10_0 | Platform.Watch_3_0, Deprecated = Platform.iOS_11_0 | Platform.Watch_4_0, Message = "Use 'Create (HKWorkoutEventType, NSDateInterval, HKMetadata)' instead.")]
+		[iOS (10, 0, message: "Use 'Create (HKWorkoutEventType, NSDateInterval, HKMetadata)' instead.")]
```

Comment:
This is wrong. I need to review the entire diff now...

Reply:
ouch :(
might be a good test to add (message is only used when deprecating or obsoleting an API)

Reply:
That is exactly how I caught it. I cherry picked this to my "remove PMCS avail" work and got:

```/Users/donblas/Programming/macios/master/xamarin-macios/src/healthkit.cs(2056,16): error CS1739: The best overloaded method match for `MonoTouch.ObjCRuntime.iOSAttribute.iOSAttribute(byte, byte, byte)' does not contain a parameter named `message'
/Users/donblas/Programming/macios/master/xamarin-macios/src/build/ios/compat/core.dll (Location of the symbol related to previous error)```

Reply:
👍

Reply:
I did a full search of the diff, and this appears to be isolated. There were 3 in close and 2 of them had one syntax and one another. My eyes my have glazed over and I hit the wrong macro three times.

This section of conversion was only semi-automated.

I also hit a number of "should be converted but weren't due to not matching my pattern". I'm fixing those as well.